Source/WebCore/platform/graphics/FontCascadeDescription.h

```diff
@@ -165,6 +167,7 @@ class FontCascadeDescription : public FontDescription {
     unsigned m_fontSmoothing : 2; // FontSmoothingMode
     // True if a web page specifies a non-generic font family as the first font family.
     unsigned m_isSpecifiedFont : 1;
+    float m_zoom { 1.0f };
```

Comment:
Shouldn't this new member affect `operator==`?

Reply:
fixed